target/linux/ath79/image/generic.mk

```diff
@@ -664,6 +664,17 @@ define Device/jjplus_ja76pf2
 endef
 TARGET_DEVICES += jjplus_ja76pf2
 
+define Device/kuwfi_cpe206
+  ATH_SOC := qca9531
+  DEVICE_VENDOR := KuWfi
+  DEVICE_MODEL := CPE206
+  DEVICE_ALT0_VENDOR = KuWfi
+  DEVICE_ALT0_MODEL = CPE2G-9531
+  DEVICE_PACKAGES += -uboot-envtools
```

Comment:
This should be a ":=" instead of a "+=" since there is no parent definition.

Reply:
done